dracut-iguana/dracut-iguana.changes

```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Fri Jan  6 13:25:44 UTC 2023 - Ondrej Holecek <oholecek@suse.com>
+
+- Add root partition auto-detection and encrypted root detection
```

Comment:
I am used to exactly opposite :) Only bugzilla numbers make it to changelog. But as this is independed project, we for sure can make it so PR# or Issue# there. I will add it.

Reply:
What about using the gh#openSUSE#6 schema? See https://en.opensuse.org/openSUSE:Packaging_Patches_guidelines#Current_set_of_abbreviations. Anyway, I do not want to block this PR because of that.

Reply:
That make sense, `gh#iguana` prefix. But I would really like to differentiate between issue and PR. Happened to me couple of times with salt project where I did the PR and then was forced to create dummy issue just pointing to the PR.

Ok with `gh#iguana#XX` for issues and `gh#iguana#prXX` for PRs?

Reply:
Oh, yes, the problem is that GitHub does not distinguish both cases. For me it is fine using `#prXX` and `#XX`. Remember that the proposal contains the "full" name of the repo: `gh#openSUSE/iguana#XX`, so you know which repo it comes from even if you do not know the project (although I know it is rather long).

Reply:
On the other hand, I have seen some tool that validates the numbers in OBS, and I am not sure it will like the `#pr` prefix :)

Reply:
We'll see. I will update contribution section with this info in next PR.